### PR TITLE
Update datasource with additional fields

### PR DESCRIPTION
**Pull request by safaci2000**
Adding additional datasource fields to match: https://grafana.com/docs/grafana/latest/http_api/data_source/

### CODE REVIEW

**Comment by safaci2000** [comment]
Recent versions of Grafana have started to link the dashboard to the datasource by UID.  I need those fields added in order to create a valid mapping.

Previous versions:

{
      "datasource": "Flow - Rollup by Edge ASNs (5m)",
      ...
}

New version:

{
      "datasource": {
        "type": "elasticsearch",
        "uid": "VuEGTpa7k"
      }
      ...
}

There might be more PRs coming but at least for now, getting the DS will persist the JSON fully.

**Comment by safaci2000** [comment]
Just a friendly poke if this can be merged or if we have any objections.

**Comment by safaci2000** [comment]
poke, poke, any thoughts or objections?

File: datasource.go [hunk 1]
```diff
@@ -24,14 +24,17 @@ package sdk
 type Datasource struct {
 	ID                uint        `json:"id"`
 	OrgID             uint        `json:"orgId"`
+	Uid               string      `json:"uid"`
```
Comment (GiedriusS) [inline datasource.go:27]: Let's call it `UID` for consistency (https://github.com/grafana-tools/sdk/blob/f7c763c3738c279f7e926c5a3bc8353b4080312b/folder.go#L26)

File: datasource.go [hunk 2]
```diff
@@ -24,14 +24,17 @@ package sdk
...
 	Name              string      `json:"name"`
 	Type              string      `json:"type"`
+	TypeLogoUrl       string      `json:"typeLogoUrl"`
```
Comment (GiedriusS) [inline datasource.go:30]:
```suggestion
	TypeLogoURL       string      `json:"typeLogoUrl"`
```
same here (https://github.com/grafana-tools/sdk/blob/e7b85bb25aa9694012fa6f829a2eed879e5a33ba/rest-request.go#L40)

**Review by GiedriusS — COMMENTED** [review]
Sorry for the delay

File: datasource.go (hunk 1 above)
Comment (safaci2000) [reply datasource.go:27]: Fixed

**Comment by safaci2000** [comment]
Addressed comments, let me know if anything else is missing.

**Review by GiedriusS — APPROVED** [review]

**Pull request merged**